sktime/forecasting/compose/_pipeline.py

```diff
@@ -1635,7 +1668,13 @@ def get_test_params(cls, parameter_set="default"):
 
         params2 = {"forecaster_X": fx, "forecaster_y": fy_proba, "behaviour": "refit"}
 
-        return [params1, params2]
+        params3 = {
+            "forecaster_y": fy,
```

Comment:
It needs a `forecaster_X`, which is a mandatory argument. At least, it needs to be passed as `None` to use `forecaster_y`.

Reply:
yes, that was a typo and the reason for the test failures.

Reply:
hm, I see, it needs both. The default should be `None`, I added that (this will not break dependent code)